snippets/csharp/System/DateTime/ToString/ToString4.cs

```diff
@@ -1,38 +1,40 @@
-﻿// <Snippet3>
-using System;
+﻿using System;
 using System.Globalization;
 
 public class ToStringExample3
 {
-   public static void Main2()
-   {
-      CultureInfo[] cultures = new CultureInfo[] {CultureInfo.InvariantCulture,
-                                                  new CultureInfo("en-us"),
-                                                  new CultureInfo("fr-fr"),
-                                                  new CultureInfo("de-DE"),
-                                                  new CultureInfo("es-ES"),
-                                                  new CultureInfo("ja-JP")};
+    public static void Main()
+    {
+        // <Snippet3>
+        CultureInfo[] cultures = [
+            CultureInfo.InvariantCulture,
+            new CultureInfo("en-us"),
+            new CultureInfo("fr-fr"),
+            new CultureInfo("de-DE"),
+            new CultureInfo("es-ES"),
+            new CultureInfo("ja-JP")
+            ];
```

Comment:
Is this the right placement of `];`?

The VS formatter looks to not handle it today, but visually speaking I think:
```csharp
CultureInfo[] cultures = [
    CultureInfo.InvariantCulture,
    new CultureInfo("en-us"),
    new CultureInfo("fr-fr"),
    new CultureInfo("de-DE"),
    new CultureInfo("es-ES"),
    new CultureInfo("ja-JP")
];
```

reads better than:
```csharp
CultureInfo[] cultures = [
    CultureInfo.InvariantCulture,
    new CultureInfo("en-us"),
    new CultureInfo("fr-fr"),
    new CultureInfo("de-DE"),
    new CultureInfo("es-ES"),
    new CultureInfo("ja-JP")
    ];
```

and it "lines up" with how other braces tend to work, where the closing brace is "aligned" with the starting line.